library.html

```diff
@@ -35,155 +35,174 @@
 			</div>
 			
 			<!--Content Boxes-->
-				<div class = "content_3">
+			<div class = "main_content">
```

Comment:
I have added a music display onto the library page that will be on top of of the music labels